src/frontend/components/form-group/calc-fields/lib/component.js

```diff
@@ -62,10 +62,12 @@ class CalcFieldsComponent extends Component {
         var returnval = func.apply(first, vars)
 
         // Update the field holding the code's value
-        $field.find('input').val(returnval)
+        $field.find('textarea').val(returnval)
```

Comment:
Is `.val()` definitely needed here?

Reply:
Correct it's no longer needed. Now Removed.